GitTfs.Vs2010/GitTfs.Vs2010.csproj

```diff
@@ -146,6 +143,6 @@
   </Target>
   -->
   <PropertyGroup>
-    <PostBuildEvent>xcopy /y /I "$(TargetDir)*.dll" "$(SolutionDir)GitTfs\$(OutDir)$(ProjectName)" /EXCLUDE:$(SolutionDir)files_to_ignore_during_post_build_copy.txt</PostBuildEvent>
+    <PostBuildEvent>xcopy /y /I "$(TargetPath)" "$(SolutionDir)GitTfs\$(OutDir)$(ProjectName)\*"</PostBuildEvent>
```

Comment:
In fact for project != Vs2015, we only need to copy the `GitTfs.Vs201x.dll` (`$(TargetPath)`) file and not all the dlls that were in the output directory. So no more need to use the `/EXCLUDE` option.

But if we vopy only one file, xcopy believe that the 2nd parameter is the destination path to the file. So we should add `\*` to tell him that's a directory and it should put it inside.